batch.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+usage() {
+	echo "batch.sh path_to_videos"
+	exit 0
+}
+
+input=$1
+script_path="$(cd "$(dirname "$0")" && pwd)"
+
+if [[ -d "$input" ]];
```

Comment:
If you’re putting `then` in the next line, then putting a `;` here doesn’t make much sense (it isn’t needed). In addition, you’re separating them here, but on the rest of the submission you keep them together. The rest of the script already in place seems to keep it together, so it should likely be done here as well.

Not that it’ll change the result, naturally, but consistency in these small things is always a good thing<sup>1</sup>.

---

<sup>1</sup>Arguable if the alternative brings a great advantage in that particular instance, without sacrificing readability.